tests/e2e/specs/stories-editor/story-templates.js

```diff
@@ -15,9 +15,59 @@ import {
 	searchForBlock as searchForStoryBlock,
 } from '../../utils';
 
+/**
+ * Adds a reusable block.
+ */
+async function addReusableBlock() {
+	await createNewPost();
+
+	const isTopToolbarEnabled = await page.$eval( '.edit-post-layout', ( layout ) => {
+		return layout.classList.contains( 'has-fixed-toolbar' );
+	} );
+	if ( ! isTopToolbarEnabled ) {
+		await clickOnMoreMenuItem( 'Top Toolbar' );
+	}
+
+	// Remove all blocks from the post so that we're working with a clean slate.
+	await page.evaluate( () => {
+		const blocks = wp.data.select( 'core/block-editor' ).getBlocks();
+		const clientIds = blocks.map( ( block ) => block.clientId );
+		wp.data.dispatch( 'core/block-editor' ).removeBlocks( clientIds );
+	} );
+
+	// Insert a paragraph block
+	await insertBlock( 'Paragraph' );
+	await page.keyboard.type( 'Reusable block!' );
+
+	await clickButtonByLabel( 'More options' );
+
+	const convertButton = await page.waitForXPath( '//button[text()="Add to Reusable Blocks"]' );
+	await convertButton.click();
+	// Wait for the snackbar confirmation of the block creation result.
+	await page.waitForSelector( '.components-snackbar__content' );
+}
+
+/**
+ * Removes all reusable blocks visible on the wp_blocks edit screen.
+ */
+async function removeAllReusableBlocks() {
+	await visitAdminPage( 'edit.php', 'post_type=wp_block' );
+
+	// Delete all reusable blocks to restore clean state.
+	const selector = '#cb-select-all-1';
+	const actionsSelector = '#bulk-action-selector-top';
+
+	await page.click( selector );
+	await page.select( actionsSelector, 'trash' );
```

Comment:
The `should hide story templates from the reusable blocks management screen` test is currently failing at this point:

`Error: failed to find element matching selector "#bulk-action-selector-top"`

Reply:
https://travis-ci.org/ampproject/amp-wp/jobs/580162773#L923-L933